remoteappmanager/db/interfaces.py

```diff
@@ -69,16 +69,17 @@ def get_user_by_name(self, user_name):
 
         Returns
         -------
-        a User-like object that the Database understands
+        account : opaque-type
+           an object that the database understands
         """
 
     @abstractmethod
-    def get_apps_for_user(self, user):
-        """ Return an iterable of ApplicationConfig for a given User
+    def get_apps_for_user(self, account):
```

Comment:
In my previous branch I did that, but think it might be too trivial to worth breaking the API that is in a release.